cypress/plugins/index.js

```diff
@@ -18,4 +18,10 @@
 module.exports = (_on, _config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
+  _on("task", {
```

Comment:
Is this cleaning up console logs?

Reply:
I added it for debugging purposes since it let me see prettier logs in my console and in CircleCI. I left it in there in case it was useful, but also happy to remove!

Reply:
Can you just add a comment for what it is/how to use it?

Reply:
For sure! Just added a comment